### PR TITLE
Text copy DAO-64

### PR DESCRIPTION
**Pull request by Rekard0**
- Goven Copy changes
- Aragon UI updates
- minor UI updates

### CODE REVIEW

File: packages/govern-console/src/containers/CreateDao/CreateDaoCollateral.tsx [hunk 1]
```diff
@@ -283,9 +282,8 @@ const CreateDaoCollateral: React.FC<{
             defaultValue={isAnyAddress}
             render={({ field: { onChange, value } }) => (
               <ContentSwitcher
-                title="Schedule execution permissions"
-                subtitle="If you want you can define the list of addresses that have permission to schedule
-                  executions in your DAO, so it is not open for anyone"
+                title="Whitelist of addresses that may schedule transactions."
+                subtitle="Limit the addresses that may schedule transactions. Caution: if these addresses are incorrect or unavailable, your DAO will be locked."
```
Comment (yuetloo) [inline packages/govern-console/src/containers/CreateDao/CreateDaoCollateral.tsx]: Caution is bold in the doc, I guess subtitle is currently as string not a react-node?

File: packages/govern-console/src/containers/CreateDao/components/SuccessAction.tsx [hunk 2]
```diff
@@ -15,19 +15,16 @@ const SuccessAction: React.FC<{
   return (
     <div>
       <StyledText name={'body2'}>
-        Your DAO is ready. Do you wanna register your token in Aragon Voice?
-      </StyledText>
-      <StyledText name={'body2'} style={{ color: theme.disabled }}>
-        This allows you create governance proposals easy with 0 gas price
+        Your DAO is ready. Next, register your token on Aragon Voice.
```
Comment (yuetloo) [inline packages/govern-console/src/containers/CreateDao/components/SuccessAction.tsx]: the doc said if the token is registered, do not show the text asking to register

File: packages/govern-console/src/containers/CreateDao/components/SuccessAction.tsx [hunk 3]
```diff
@@ -15,19 +15,16 @@ const SuccessAction: React.FC<{
...
       </StyledText>
+
       {isNewDaoTokenRegistered ? (
         <div>
           <Button
             size={'large'}
             mode={'primary'}
             style={{
               marginTop: 20,
-              marginLeft: layoutName !== 'small' ? SPACING[layoutName] : '0px',
```
Comment (yuetloo) [inline packages/govern-console/src/containers/CreateDao/components/SuccessAction.tsx:30]: button text should be `DAO is ready, go to ...`

File: packages/govern-console/src/containers/CreateDao/CreateDaoBasicInfo.tsx [hunk 4]
```diff
@@ -147,7 +156,7 @@ const CreateDaoBasicInfo: React.FC<{
               </GridItem>
             </Grid>
             <Info mode={'warning'} title={''}>
-              The created token will use {tokenDecimals} decimals. For the amount, Don't append 0's.
+              Please, follow the format - 10.0 → Include decimals, e.g. 10.0
```
Comment (yuetloo) [inline packages/govern-console/src/containers/CreateDao/CreateDaoBasicInfo.tsx]: the text here should be `Include decimals ...`, without the front part (i.e. without `Please, follow the format`...

File: packages/govern-console/src/containers/CreateDao/components/SuccessAction.tsx [hunk 5]
```diff
@@ -36,19 +33,29 @@ const SuccessAction: React.FC<{
...
             }}
             onClick={() => tokenRegister()}
           >
-            Yes, register token
+            Registering token on Aragon Voice
+          </Button>
```
Comment (yuetloo) [inline packages/govern-console/src/containers/CreateDao/components/SuccessAction.tsx:48]: button text should be 'Register token'

File: packages/govern-console/src/containers/CreateDao/components/SuccessAction.tsx [hunk 6]
```diff
@@ -36,19 +33,29 @@ const SuccessAction: React.FC<{
...
+            size={'large'}
+            mode={'secondary'}
+            style={{
+              marginTop: 20,
+              marginLeft: layoutName === 'large' ? SPACING[layoutName] : '0px',
+            }}
+            onClick={goToDaoPage}
+          >
+            Don't register token
           </Button>
```
Comment (yuetloo) [inline packages/govern-console/src/containers/CreateDao/components/SuccessAction.tsx]: Button text should be 'Skip'

File: packages/govern-console/src/containers/CreateDao/components/SuccessAction.tsx [hunk 7]
```diff
@@ -36,19 +33,29 @@ const SuccessAction: React.FC<{
         </div>
       ) : (
         <div>
-          <Button size={'large'} mode={'secondary'} style={{ marginTop: 20 }} onClick={goToDaoPage}>
-            Don't register token
-          </Button>
+          <StyledText name={'body2'} style={{ color: theme.disabled }}>
+            Voice enables gasless (non-binding) governance proposals and votes.
+          </StyledText>
```
Comment (yuetloo) [inline packages/govern-console/src/containers/CreateDao/components/SuccessAction.tsx:38]: the text with the color theme.disabled is almost impossible to read, is this intentional?

File: packages/govern-console/src/containers/CreateDao/CreateDaoProgress.tsx [hunk 8]
```diff
@@ -248,7 +248,7 @@ const CreateDaoProgress: React.FC<{
     <ProgressComponent
       image={progressImage}
       title={'Creating your DAO'}
-      subTitle={'Hold tight your transaction is under process'}
+      subTitle={'Your transaction is in progress.'}
       progressList={progressList}
       info={'Please be patient and do not close this window until it finishes.'}
       action={action}
```
Comment (yuetloo) [inline packages/govern-console/src/containers/CreateDao/CreateDaoProgress.tsx:259]: I see the info `Please be patient...` even after the DAO is created.. is this intentional?

Comment (yuetloo) [reply packages/govern-console/src/containers/CreateDao/CreateDaoProgress.tsx:259]: missed another change for `Register token on Aragon voice → Registering token on Aragon Voice.`

File: packages/govern-console/src/containers/CreateDao/CreateDaoBasicInfo.tsx (hunk 4 above)
Comment (yuetloo) [reply packages/govern-console/src/containers/CreateDao/CreateDaoBasicInfo.tsx]: To be consistent, should we update the validateAmountForDecimals() in validations.tsx with the same format message as well? It's used in the transactions dialog..

**Review by yuetloo — CHANGES_REQUESTED** [review]

File: packages/govern-console/src/containers/CreateDao/CreateDaoCollateral.tsx (hunk 1 above)
Comment (Rekard0) [reply packages/govern-console/src/containers/CreateDao/CreateDaoCollateral.tsx]: done

File: packages/govern-console/src/containers/CreateDao/components/SuccessAction.tsx (hunk 2 above)
Comment (Rekard0) [reply packages/govern-console/src/containers/CreateDao/components/SuccessAction.tsx]: done

File: packages/govern-console/src/containers/CreateDao/components/SuccessAction.tsx (hunk 3 above)
Comment (Rekard0) [reply packages/govern-console/src/containers/CreateDao/components/SuccessAction.tsx:30]: done

File: packages/govern-console/src/containers/CreateDao/CreateDaoBasicInfo.tsx (hunk 4 above)
Comment (Rekard0) [reply packages/govern-console/src/containers/CreateDao/CreateDaoBasicInfo.tsx]: first comment: done.
second comment: yes, make sense, done

File: packages/govern-console/src/containers/CreateDao/components/SuccessAction.tsx (hunk 5 above)
Comment (Rekard0) [reply packages/govern-console/src/containers/CreateDao/components/SuccessAction.tsx:48]: done

File: packages/govern-console/src/containers/CreateDao/components/SuccessAction.tsx (hunk 6 above)
Comment (Rekard0) [reply packages/govern-console/src/containers/CreateDao/components/SuccessAction.tsx]: done

File: packages/govern-console/src/containers/CreateDao/CreateDaoProgress.tsx (hunk 8 above)
Comment (Rekard0) [reply packages/govern-console/src/containers/CreateDao/CreateDaoProgress.tsx:259]: first comment: it was not in the scope of this task, I asked team leader and it is now is include.
secon comment: done

File: packages/govern-console/src/containers/CreateDao/components/SuccessAction.tsx (hunk 7 above)
Comment (Rekard0) [reply packages/govern-console/src/containers/CreateDao/components/SuccessAction.tsx:38]: nice catch, it should be theme.disabledContent

**Review by yuetloo — APPROVED** [review]

**Pull request merged**